recipes/pysdl2-dll/meta.yaml

```diff
@@ -15,16 +15,17 @@ build:
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
-  host:
-    - pip
-    - python
-  run:
-    - python
+  build:
     - sdl2  # [osx]
     - sdl2_gfx  # [osx]
     - sdl2_image  # [osx]
     - sdl2_mixer  # [osx]
     - sdl2_ttf  # [osx]
+  host:
```

Comment:
this package vendors SDL2.

We should patch this to be an empty package and depend (at runtime) on the sdl2 packages on conda-forge.